base_running/app/controller.py

```
@@ -0,0 +1,63 @@
+
+# --- API Router ---
+# API 엔드포인트를 정의하는 라우터입니다.
+router = APIRouter(
+    prefix="/items",  # 이 라우터의 모든 경로는 "/items"로 시작합니다.
+    tags=["Items"],   # API 문서에서 "Items" 태그로 그룹화됩니다.
+)
+
+@router.post("/", response_model=schemas.Item, status_code=201)
+def create_item_endpoint(
+    item: schemas.ItemCreate,
+    svc: service.ItemService = Depends(get_item_service)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)